### DISCUSSION TITLE
VTK Book - Markdown version of the Visualization Toolkit Book

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#5117] [Show and tell]
I found that [VTK Book - Markdown version of the Visualization Toolkit Book ](https://github.com/Kitware/vtk-book) repository is awesome. VTK Book is available in markdown, let's respect the license and take advantage of it.

**Comment by banesullivan** [comment]
It would be great to at a minimum, cross-reference from PyVista's docs to this version of the VTK Textbook. I wonder if this would be possible with [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html).

For example, PyVista has a whole [guide to data types](https://docs.pyvista.org/version/stable/user-guide/what-is-a-mesh.html) and we often refer to the VTK textbook. It would be helpful to provide direct links for users to learn more

cc @jcfr
